docs/guides/csp-nonce.md

```diff
@@ -80,7 +80,7 @@ This is an example of consuming the `dist-custom-elements` output in an Angular
 // main.ts
 
 import { defineCustomElements, setNonce } from 'my-lib/dist/components';
-// Assume `autoDefineCustomElements` is enabled enabled
```

Comment:
Since this is for v3.1+, shouldn't the changes in the v3.0 docs be applied here too?

Reply:
Oh gee, yes, thank you. Gotta get used to this new docs versioning thing 🤦